help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -135,35 +140,38 @@ Read [Activate profiles and segments to streaming segment export destinations](/
 
-1. To map any custom attributes, select target field window, select the target field and choose the **[!UICONTROL Select attributes]** > **Email Demographics** category. Next provide the desired target attribute name and add the mappings desired.
-![Target mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/target-mapping-custom.png)
+    * Add the following mapping between your XDM profile schema and your [!DNL Salesforce Marketing Cloud] instance:
+        |XDM Profile Schema|[!DNL Salesforce Marketing Cloud] Instance| Mandatory|
+        |---|---|---|
+        |`contactKey`|`salesforceContactKey`| Yes |
 
-1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Salesforce Marketing Cloud] instance:
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!DNL Salesforce REST API] documentation](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts) is given below. Also note that the destination uses the [Salesforce Search Attribute-Set Definitions REST API](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/retrieveAttributeSetDefinitions.html) to retrieve attributes defined within Salesforce for your contacts and specific to your account.
+    ![Platform UI screenshot showing Target mapping.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/target-mapping-custom.png)
 
-    ||XDM Profile Schema|[!DNL Salesforce Marketing Cloud] Instance| Mandatory|
-    |---|---|---|---|
-    |Attributes|<ul><li><code>person.name.firstName</code></li><li><code>personalEmail.address</code></li></ul>|<ul><li><code>Email Demographics.First Name</code></li><li><code>Email Addresses.Email Address</code></li></ul>|<ul><li>-</li><li><code>Yes</code></li></ul>|
-    |Identities|<ul><li><code>contactKey</code></li></ul>|<ul><li><code>salesforceContactKey</code></li></ul>|Yes|
+    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Salesforce Marketing Cloud] instance:
+        |XDM Profile Schema|[!DNL Salesforce Marketing Cloud] Instance|
+        |---|---|
+        |`person.name.firstName`|`Email Demographics.First Name`|
+        |`personalEmail.address`|`Email Addresses.Email Address`|
 
-1. An example using these mappings is shown below:
-![Target mapping LastName](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/mappings.png)
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/mappings.png)
 
 ### Schedule segment export and example {#schedule-segment-export-example}
 
```

Comment:
Comment applies to row 178 just below. To give the reader a clearer picture of what they need to do, let's add a link to the section above which talks about creating custom attributes in Salesforce, like below:

`you must manually map Platform segments to the [custom attribute](#prerequisites-custom-field) in Salesforce.`